docs/release-notes/NuGet-6.5.md

```diff
@@ -0,0 +1,73 @@
+---
+title: NuGet 6.5 Release Notes
+description: Release notes for NuGet 6.5 including new features, bug fixes, and DCRs.
+author: <TODO: GitHubAlias>
+ms.author: <TODO: MicrosoftAlias>
+ms.date: 2/9/2023
+ms.topic: conceptual
+---
+
+# NuGet 6.5 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.5**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.5](https://visualstudio.microsoft.com/downloads/) | [<TODO: Full SDK Version>](https://dotnet.microsoft.com/download/dotnet-core/<TODO: SDKMajorMinorVersionOnly)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.5
+
+* Machine readable output for dotnet list package - [#7752](https://github.com/NuGet/Home/issues/7752)
+
+* Honor WarningsNotAsErrors - [#5375](https://github.com/NuGet/Home/issues/5375)
+
+* Reload Visual Studio package sources when nuget.config is modified manually - [#1538](https://github.com/NuGet/Home/issues/1538)
+
+### Issues fixed in this release
+
+**DCRs:**
+
+* During MSBuild project SDK resolution, always log errors in the case of failure - [#12312](https://github.com/NuGet/Home/issues/12312)
+
+**Bugs:**
+
+* StackOverflow in SemanticVersion.ToString - [#12330](https://github.com/NuGet/Home/issues/12330)
```

Comment:
Due to our build and insertion pipeline issues, this didn't make it into 6.5. I just changed the issue's milestone to 6.6. Please remove it from the 6.5 release notes.